### ISSUE TITLE
[SUGGESTION] Set Nameplate distance to 41 yards

### ISSUE THREAD

**Issue by DBFBlackbull** [balakethelock/SuperWoW#33]
**Is your feature request related to a problem? Please describe.**
I am frustrated that I cannot see which spells mobs or players are casting when they are at 30 - 40 yards and I do not have them as my primary target.

**Describe the solution you'd like**
I would like to be able to show nameplates of mobs that are further away than 20 yards. This is done with the https://github.com/brndd/vanilla-tweaks modification.

> **Nameplate range change**
> - Increased to 41 yards to match the maximum value in Classic and TBC Classic. 20 yards is the default value.

I have previously used vanilla tweaks, which works well, but I using that I miss the feature from Superwow that allows me to see which mob cast which spell

**Describe alternatives you've considered**
If there was a way to load Vanillatweaks together with superwow, then that would be awesome! I don't know if any of the features from the two dlls would overwrite each other in a destructive manner, so I would vastly prefer if superwow could get this feature added.

**Additional context**
Here is two screenshots. The first is Superwow at 25 yards, maximum charge distance.
![WoWScrnShot_101124_232856](https://github.com/user-attachments/assets/1d65e1f8-839a-42e0-9dd0-52bd48e37ee6)
The second is vanillatweaks at 30 yards (you can see my charge is now red and outside of range) and I can still see the nameplates
![WoWScrnShot_101124_233026](https://github.com/user-attachments/assets/7475951a-7918-4eaf-8294-422bbb8e01b7)

**Comment by balakethelock** [comment]
> If there was a way to load Vanillatweaks together with superwow, then that would be awesome!
Already possible

**Comment by DBFBlackbull** [comment]
That sounds awesome! I have looked at the SuperAPI addon but I was not able to find any option to control this. I have also tried various `/console` commands, but all without luck.

I never said that vanillatweaks was a dll.

Could you please provide a guide as to how I would go about getting my nameplates to 41 yards using superwow? I have searched the net and haven't been able to find a guide using the two together.

What I have tried is renaming my `WoW_tweaked.exe` to `WoW.exe` and then use SuperwowLauncher, but when I try to log in I get the following error:
![image](https://github.com/user-attachments/assets/fb81aeab-bc34-4bd8-a925-e7054e401f9a)

So any guide on how to continue would be awesome! Thank you!

**Comment by balakethelock** [comment]
Do you get this login error only when you use superwowLauncher with a tweaked & renamed exe, or does it happen also when you run vanillatweaks alone

**Comment by DBFBlackbull** [comment]
When I run vanilla_tweaks.exe alone everyone works fine.

When I rename the original .exe to `WoW-original.exe` and then rename `WoW_tweaked.exe` to `WoW.exe` then I get the error no matter what I do.

**Comment by DBFBlackbull** [comment]
I did mange to get it working through!

I used both `WoW_tweaked.exe` `SuperWow.dll` and `vanillafixes.exe`

Here are my steps.

1. Copy both Superwow files and vanillafixes files into the wow directory.
2. Drag the `WoW_tweaked.exe` onto the `vanillafixes.exe`. This starts the game
3. Now the login works.

It seems that my core issue was that if the original `WoW.exe` got renamed to anything, then the client would not allow login, so keeping everything with their orignal names was the trick!
